.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I am opposed to committing IDE specific files into Git. Instead `.vscode/` should be globally ignored in your `$HOME/.gitignore`, reasoning for this is about the same as [what I wrote here](https://github.com/fluxcd/flux/pull/2546#issuecomment-545838516).

Reply:
oh, that was not intentional. deleted